### PR TITLE
Cleaning up/restructuring

### PR DESCRIPTION
**Pull request by sarah-johnson**

### CODE REVIEW

File: flask_api/static/page_data/story/storybooks/storybooks.md [hunk 1]
```diff
@@ -1 +0,0 @@
-# Storybooks
```
Comment (sarah-johnson) [inline flask_api/static/page_data/story/storybooks/storybooks.md:1]: You get the title for free and it's okay to have a yaml page defined with no markdown so I just removed the stubs which produced duplicate titles.

File: static/page_data/rulebook/magic/magic.md [hunk 2]
```diff
@@ -4,19 +4,19 @@ The world of Karst is a magical one. Characters can learn to draw power from or
 
 The cosmology of Karst has spawned four distinct types, or Forms, of magic, Thaumaturgy, Theurgy, Spiritualism, and Deceit. A Practitioner of Magic is the term for anyone who has the Knack for one or more of these Forms. Becoming versed in any of these mysteries is possible for any character, but may prove incredibly difficult or involve sacrifice and/or devotion. 
 
-## [Deceit](Deceit)
+## [Deceit](deceit)
```
Comment (sarah-johnson) [inline static/page_data/rulebook/magic/magic.md:7]: We could make these not-case-sensitive but this makes the links work for now.

Comment (sarah-johnson) [reply static/page_data/rulebook/magic/magic.md:7]: They "work" but they don't work the same way that react-router works.  They force the whole application to refresh, API to be re-queried, and it also causes a "page %s not found" to flash across the screen although it eventually resolves.

There must be an MDX way to make these react-router Links?

Comment (vonkow) [reply static/page_data/rulebook/magic/magic.md:7]: I think if we just write them as react-router <Link>s in the MDX and add the Link component via a provider (I saw something about this in the MDax docs) then they should work. I’ll try it out in a bit

**Pull request merged**